input/script_utils.go

```diff
@@ -35,7 +35,7 @@ type Signature interface {
 // WitnessScriptHash generates a pay-to-witness-script-hash public key script
 // paying to a version 0 witness program paying to the passed redeem script.
 func WitnessScriptHash(witnessScript []byte) ([]byte, error) {
-	bldr := txscript.NewScriptBuilder()
+	bldr := txscript.NewScriptBuilder(txscript.WithScriptAllocSize(34))
```

Comment:
Perhaps consider adding named constant to represent the integer 34? And I think the same goes for the other changes in this file. We could then use these constants in the new (benchmarking) tests.